### PR TITLE
[Android]Change the name of javascript object used in instrumentation tests

### PR DESCRIPTION
**Pull request by minggangw**
Rename the original js object echo to echoJava to indicate that the object
stands for extensions written in Java. Also, the corresponding html files
used for testing are changed to echo_java.html/echo_sync_java.html
    
The purpose of renaming is to avoid conflicting if the extension is written
 in C++. We need a unique name to differentiate these objects.

### CODE REVIEW

**Comment by minggangw** [comment]
Open a new PR for the naming issue, @halton @wuhengzhi please review it, thanks.

File: test/android/core_internal/javatests/src/org/xwalk/core/internal/xwview/test/ExtensionEchoInternalTest.java [hunk 1]
```diff
@@ -33,7 +33,7 @@ public void testAsync() throws Throwable {
     public void testSync() throws Throwable {
         ExtensionEchoInternal echo = new ExtensionEchoInternal();
 
-        loadAssetFile("echoSync.html");
+        loadAssetFile("echoSync_java.html");
```
Comment (halton) [inline test/android/core_internal/javatests/src/org/xwalk/core/internal/xwview/test/ExtensionEchoInternalTest.java]: Please rename it as `echo_sync_java.html`, it is your not fault, I'd like to make it right to follow chromium coding style.

Comment (minggangw) [inline test/android/core_internal/javatests/src/org/xwalk/core/internal/xwview/test/ExtensionEchoInternalTest.java]: No problem :)

File: runtime/android/sample/src/org/xwalk/core/sample/ExtensionEcho.java [hunk 2]
```diff
@@ -9,7 +9,7 @@
 public class ExtensionEcho extends XWalkExtension {
 
     public ExtensionEcho() {
-        super("echo",
+        super("echo_java",
```
Comment (halton) [inline runtime/android/sample/src/org/xwalk/core/sample/ExtensionEcho.java]: Sorry, I did not make it clear. The file name should be like `foo_bar`. The Java/JavaScript variable name should be like `fooBar`

Please refer to the chromium coding style page at https://www.chromium.org/developers/coding-style

**Comment by halton** [comment]
LGTM, will merge after trybot get green.

Many thanks for @minggangw 's patient rework.

**Pull request merged**